geomesa-fs/geomesa-fs-tools/src/main/scala/org/locationtech/geomesa/fs/tools/ingest/StorageJobUtils.scala

```diff
@@ -49,4 +47,29 @@ object StorageJobUtils extends LazyLogging {
     }
     success
   }
+
+  private def distCpOptions(src: Path, dest: Path): DistCpOptions =
+    try { distCpOptions3(src, dest) } catch { case _: ClassNotFoundException => distCpOptions2(src, dest) }
```

Comment:
Should we have debug/trace logging around this try and fallback approach?

Reply:
mmm, we could i guess... would anyone ever care though unless it failed? it's only called once on a m/r job configure, so i think the performance penalty from throwing the exception should be negligible.

Reply:
Meh, works for me.